cl/search/management/commands/cl_update_index.py

```diff
@@ -299,6 +310,22 @@ def add_or_update(self, *items):
             add_or_update_audio_files.delay(items)
 
     @print_timing
+    def add_or_update_all_recap_files(self):
+        """
+        Updates The index of all the dockets whose source is RECAP.
+        :return:
+        """
+        self.stdout.write("Adding or updating all RECAP files.")
+        # Doing the updating asynchronously.
+        if self.type == Docket:
+            self.stdout.write("Adding or updating all RECAP dockets.")
+            pk_list = []
+            for docket in Docket.objects.filter(source=Docket.RECAP):
+                pk_list.append(docket.pk)
+            add_or_update_recap_dockets.delay(pk_list)
```

Comment:
This will have bad performance. It'll pull all columns for the entire queryset into memory.